compiler/luci/profile/src/CircleNodeOrigin.cpp

```diff
@@ -141,7 +141,13 @@ namespace luci
 
 bool has_origin(const luci::CircleNode *circle_node)
 {
-  return circle_node->annot<CircleNodeOriginAnnotation>() != nullptr;
+  if (circle_node->annot<CircleNodeOriginAnnotation>() == nullptr)
+    return false;
+
+  if (circle_node->annot<CircleNodeOriginAnnotation>()->origin() == nullptr)
+    return false;
```

Comment:
When does this condition happen?

Reply:
Let's assume that a model has no metadata.
Then `nullptr` will be propagated in `luci/pass`.
In that situation, #6101 has following codes.

https://github.com/Samsung/ONE/blob/d9a735e88c6be93507d17cb08c395fe0580a1f9b/compiler/luci/export/src/CircleOperationExporter.cpp#L1497-L1505

If the logic is not added, `get_origin` returns `nullptr` and it causes SEGFAULT.